modules/web/src/app/settings/admin/seed-configurations/component.ts

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnDestroy, ViewChild} from '@angular/core';
+import {Router} from '@angular/router';
+import {forkJoin, of, Subject} from 'rxjs';
+import {switchMap, takeUntil} from 'rxjs/operators';
+import {MatTableDataSource} from '@angular/material/table';
+import {DatacenterService} from '@core/services/datacenter';
+import {SeedOverview} from '@shared/entity/datacenter';
+import {MatSort} from '@angular/material/sort';
+import _ from 'lodash';
+import {handleSeedOverviewDatasource, SeedOverviewDatasource} from '@shared/utils/seed-configurations';
+
+enum Column {
+  Phase = 'phase',
+  Name = 'name',
+  Providers = 'providers',
+  Datacenters = 'datacentersCount', // # total datacenters per seed
+  Clusters = 'clustersCount', // # total datacenters per seed
```

Comment:
i think the comment is `total clusters for all datacenters`

Reply:
per/seed = sum of all datacenters isn't it?